apps/microsoft-teams/frontend/src/components/config/AccessSection/AccessSection.tsx

```diff
@@ -38,6 +42,14 @@ const AccessSection = (props: Props) => {
   const { instance, accounts, inProgress } = useMsal();
   const customApi = useCustomApi();
   const sdk = useSDK<ConfigAppSDK>();
+  const { trackEvent } = useContext(SegmentAnalyticsContext);
+  const trackingEventInfo = {
+    environment_key: sdk.ids.environment,
+    space_key: sdk.ids.space,
+    organization_key: sdk.ids.organization,
+    app_key: sdk.ids.app,
+    app_name: 'microsoft-teams',
+  };
```

Comment:
I added this object to resolve the type errors - it looks like the Segment schema is expecting this information.